### PR TITLE
Transparent ssao

### PR DESCRIPTION
**Pull request by giagitom**
# Description

### **Implement transparent ssao**

1. Added options to toggle ssao on transparency / opacity
2. Added option to toggle single / double texture approach 

image

## Actions

- [x] Added description of changes to the `[Unreleased]` section of `CHANGELOG.md`
- [x] Updated headers of modified files
- [ ] Added my name to `package.json`'s `contributors`
- [ ] (Optional but encouraged) Improved documentation in `docs`

### CODE REVIEW

File: src/mol-gl/shader/postprocessing.frag.ts [hunk 1]
```diff
@@ -70,6 +79,7 @@ float getOutline(const in vec2 coords, const in float opaqueDepth, out float clo
             if (sampleOutline == 0.0 && sampleOutlineDepth < closestTexel) {
                 outline = 0.0;
                 closestTexel = sampleOutlineDepth;
+                isTransparent = sampleOutlineCombined.a;
```
Comment (arose) [inline src/mol-gl/shader/postprocessing.frag.ts:82]: is that a general fix for which you can do a separate PR?

**Comment by arose** [comment]
thank you @giagitom! will need some time to look at it closely

File: src/mol-gl/shader/postprocessing.frag.ts (hunk 1 above)
Comment (giagitom) [reply src/mol-gl/shader/postprocessing.frag.ts:82]: This is just useful with the new postprocessing pass configuration. Since opaque and transparent color blending is done within the postprocessing fragment it is now possible to apply opaque and transparent outlines separately similarly to ssao

File: src/mol-gl/shader-code.ts [hunk 2]
```diff
@@ -170,11 +170,14 @@ function ignoreDefine(name: string, variant: string, defines: ShaderDefines): bo
...
         const ignore = [
             'dColorType', 'dUsePalette',
-            'dLightCount', 'dXrayShaded',
             'dOverpaintType', 'dOverpaint',
             'dSubstanceType', 'dSubstance',
-            'dColorMarker', 'dCelShaded'
+            'dColorMarker', 'dCelShaded',
+            'dLightCount',
         ];
+        if (variant !== 'depth') {
```
[the remark this answers is not in the document]
Comment (giagitom) [reply src/mol-gl/shader-code.ts:178]: This fixed the issue I mentioned to you privately of XRayShaded objects not being occluded. If necessary I'll make a separate PR for this. Let me know

**Comment by arose** [comment]
Thank you @giagitom for your initiative and patience resolving my concerns. It is a very nice improvement!
